### ISSUE TITLE
b r u h

### ISSUE THREAD

**Issue by oCold** [axtloss/pear-no-s#2]
so , as far im concerned you are really getting to mans ass

1.there is nothing wrong of pearos being based on KDE Neon lmao and the install breaks yes , there is no other option as long pearOS is supposed to be a linux macos clone distro except its not indepedent distro

2. its not the first time u "whine" about piri , like if you are so desperated to make pearos stop using then take it down why u gotta make a whole drama

3 the way how u send people to pearOS server to give you infos basically snitching is very funny at all because u have no courage to show up yourself

4 ofc pearOS has slow support , because you totally know there is only one person developing it 

5 this whole repo u made on github , it wasnt to help people but to hurt alex reputation which is very low movement and shows what a ego you have

6 not here to protect alex or you , both of you are guilty(alex about the piri stuff) im just saying how i see the situation in my perspective and trying to say u are doing wrong

**Comment by axtloss** [comment]
> there is nothing wrong of pearos being based on KDE Neon
I never said there is.

> its not the first time u "whine" about piri ,
yeah you're correct, except I'm not "whining" about it this time, I'm just saying what happened.

> if you are so desperated to make pearos stop using then take it down why u gotta make a whole drama
ah yes, take down my program so that someone stops using it, epic idea.

> the way how u send people to pearOS server
I have never sent people to the discord to show me screenshots, the stuff I show is just stuff I picked up while seeing other talk about it (and sometimes asking some extra questions about it)

>  have no courage to show up yourself
oh no I would definetly show up myself, but I think I'm banned and I also don't see any reason to join a server owned by someone who's a huge bigot

> ofc pearOS has slow support
I'm not sure when I mentioned slow support, but ok

> it wasnt to help people
oh no it fully was, I made it to show what type of person alex really is and make them question if they really want to use it.

**Comment by prys4pa** [comment]
> so , as far im concerned you are really getting to mans ass
> 
> 1.there is nothing wrong of pearos being based on KDE Neon lmao and the install breaks yes , there is no other option as long pearOS is supposed to be a linux macos clone distro except its not indepedent distro
> 
>     2. its not the first time u "whine" about piri , like if you are so desperated to make pearos stop using then take it down why u gotta make a whole drama
> 
> 
> 3 the way how u send people to pearOS server to give you infos basically snitching is very funny at all because u have no courage to show up yourself
> 
> 4 ofc pearOS has slow support , because you totally know there is only one person developing it
> 
> 5 this whole repo u made on github , it wasnt to help people but to hurt alex reputation which is very low movement and shows what a ego you have
> 
> 6 not here to protect alex or you , both of you are guilty(alex about the piri stuff) im just saying how i see the situation in my perspective and trying to say u are doing wrong

lick alex's balls some more

**Comment by oCold** [comment]
> > so , as far im concerned you are really getting to mans ass
> > 1.there is nothing wrong of pearos being based on KDE Neon lmao and the install breaks yes , there is no other option as long pearOS is supposed to be a linux macos clone distro except its not indepedent distro
> > ```
> > 2. its not the first time u "whine" about piri , like if you are so desperated to make pearos stop using then take it down why u gotta make a whole drama
> > ```
> > 
> > 
> >     
> >       
> >     
> > 
> >       
> >     
> > 
> >     
> >   
> > 3 the way how u send people to pearOS server to give you infos basically snitching is very funny at all because u have no courage to show up yourself
> > 4 ofc pearOS has slow support , because you totally know there is only one person developing it
> > 5 this whole repo u made on github , it wasnt to help people but to hurt alex reputation which is very low movement and shows what a ego you have
> > 6 not here to protect alex or you , both of you are guilty(alex about the piri stuff) im just saying how i see the situation in my perspective and trying to say u are doing wrong
> 
> lick alex's balls some more

lick amys ass more dogshit

**Comment by oCold** [comment]
> also, you seem to have created this account just for opening this issue, assuming you have a proper account you use, I wouldn't talk about courage stuff

i dont use github  + didnt create it today so cope more

**Comment by oCold** [comment]
exciting to see how childrens are gonna attack me now 💀💀💀

**Comment by axtloss** [comment]
> > also, you seem to have created this account just for opening this issue, assuming you have a proper account you use, I wouldn't talk about courage stuff
> 
> i dont use github + didnt create it today so cope more

So you're not going to defend yourself on any other points?
If not I'll just close this since you seem to have stated your points and I stated mine

**Comment by alxb421** [comment]
# Start of the
## I won t be like Axtlos and I will put the racial slurs against black people at first.

![F600658C-D38F-4DE5-84F2-D8A1AA629399](https://user-images.githubusercontent.com/72302254/153033089-d2a5d63f-6032-4d4b-a715-cd0fb14e1a5a.png)

![D406D21A-36BB-46B1-8BF5-546C899D7B23](https://user-images.githubusercontent.com/72302254/153033112-10f5d82d-3d44-4127-a5b6-8f40e0e24ec8.png)

# Message for amy:

**I would not talk about the nword, "amy"**
I asked for your permission "axtlos, are you a boy?" you replied with yes. Then I asked you if it's OK to laugh at "offensive jokes" with you. I will put the conversation down in a quote ( the conv. is not 100% took from the real convo, since I don t remember the EXACT terms, but the main message is available)

You agreed with me to say/use the nword with JOKE purposes ( meaning that no racism is intended) you also said yes:
```
 alex:  are you a boy?
 amy(axtlos): yes
 alex: are you black?
 amy(axtlos) no 🤣 
 ```
This was an example of convos you had with me. Did you telll me that you are trans? no
Were you trans at the moment when I laughed at offensive memes? No
Did you say the nword AS A JOKE? yes
After that I explained you this:: 

```
If a black guy would need my help ofc I would help him.
In Balkans we don t share the same racism obsession, for Balkan people it is JUST a word
I will NOT say the nword publicly because I respect others and if someone(including you) feel offended about me saying offensive things, tell me, and I will stop
```

# The word "shemale" used on the Piri backend:
 - I thought that "shemale" is a meme word cause I seen it in many memes.
 - I showed you those memes, I removed the offensive word.
 - You understood me for not knowing that the word is offensive


# "he did not pay the CDN". 
I have to say few stuff bout this:
- 1. Why did you assume my gender? (that question is with humouristic purposes, I go by hell/boy. Jk again, I go as he/him. For real
- 2. Why do you care so much bout my CDN?? Even if it's 2EUR (1.5GBP)
- 3. You said that "They had money to pay cause they paid after <provider> said that alexb will be suspended"
  - the thing is I asked for 2 USD from my webdesigner (Andrei Muntean) (see screenshots below)
![image](https://user-images.githubusercontent.com/72302254/153035400-79b1e8a0-aade-403a-82aa-70a7f80d99dd.png)
  - I ts not quite ethical to show you this, but here is my paypal balance:
  
   ![image](https://user-images.githubusercontent.com/72302254/153035524-7c3507a2-a036-4baf-bdf6-24a603279007.png)
  
![image](https://user-images.githubusercontent.com/72302254/153036115-54f9ed8a-4d09-4541-9991-024885ef363e.png)

# As for the Neofetch thing:
I removed your name from `neofetch` command because people though that I made pearOS WITH YOU, so it was misunderstanding. (i.e. Who made pearOS? Alexandru Balan and (w/) axtlos )
I found it resonable to remove your name to stop the misunderstanding.
The copyright is owned by the Neofetch dev, and the logos authors(pear, gentoo, ubuntu, arch, pop, redstar, debian, and more)
Every logo belongs to it's author, and what you did was: get the logo from me, get neofetch from github.

# Another thing:
- This guy commented bad about me, and the comment was left as it is, but when someone defended me, you minimized the comment. This shows what a stupid behavoiur you have.
![image](https://user-images.githubusercontent.com/72302254/153047962-16496c26-2d8a-4f9e-a6b7-7109a90c1542.png)
(unhidden commenet)

![image](https://user-images.githubusercontent.com/72302254/153048050-8b2aac55-36db-4fcc-969c-0f01a79fb02d.png)
Hidden comment (was against amy right?)

This shows your unbalanced behaviour.

# Yet another thing:
- We also both agreed that neither if us is against other's sexuality as long as "they don t kiss in front of us" (you should remember this phrase that you agreed with.

# Small personal attack:
- As I can see, afrer you switched gender, you switched whole personality and behavoir, acting now like the most basic 15yo kid ever.

# My opinion 'bout LGBT people and black people

- Shortly: 
  -I do no care. My services provider (`jusuf#8562`) is trans and I do not have anything against her for being trans.
- I have black friends. If I was really a racist, I would not have black friends.
- I have friends part of the LGBT (mostly bisexuals and tomboys). Do I have anything against them? nop. They my friends, and I do not care bout their sexuality/skin color/religion
- Yes, I find cringe putting LGBT flag everywhere, because it is not something normal to do ( in my honest opinion). If you are part of LGBT, alright, good for you, nobody else cares about that. If I put "straight" in my instagram bio, I would be cancelled by the LGBT people, but if people from LGBT put the rainbow flasg in bio, they are somehow "supperior"
- I know a lot of LGBT people that are actually really funny, good, and normal(not putting "ayo I m gay" everywhere they go". I appreciate this "silent" kind of behaviour.
- For the black people: here is a good example. Give me ONE reason for hating someone based on skin color. It s like someone hating on idk, tall people? It s fucking normal, it s in human nature to be tall, short, white, asian, beautyful, ugly. I hate people based on their personality. I am extremly "racist" in this thing, meaning that I hate **all people** that **steal**, **rape**, **kill**, **lie**, **make fun of someone**, **cheat**. These are the type of people I hate.

# TL;DR: 
```
We both made the same mistakes, but the thing is that you care too much about pearOS and you are making a full drama trying to "cancel" me.
More things you did: care too much about my financial situation. Why? Is your financial stuation?
```

# Ending the
I publicly said in my server:
```
I support every Linux distro. Told that in my server too. Screenshot below.
It is EXTREMLY stupid making a gh repo saying not to use a linux distro

You DO NOT deserve being a distro developer if you say things like "don t use this distro", "that distro is bad", "ew <redacted> distro". Extremly bad and this shows what kind of person you are.

Wanted to say lot more things bout your past behaviour, but you were a different person right?

I understand you, you are a kid, you are only 15( I told you happy birthday in June, so I kinda cared bout you), while I was expecting the same from you.

Finally, keep your GenZ shit away from me, kid
```

![image](https://user-images.githubusercontent.com/72302254/153048960-2a0bb041-7df4-4f10-9d67-927cdb7e8b27.png)


# Real end of the
- You still have my best regads, since I cannot put my brain agains you, being 4 years < me
- I hope you won t remove the comment(or minimize/hide it) ad you did in (one of screenshots I sent)
- Hope you don t take me "in a bad name", I was just prooving my points without any harm intended.
- Goodbye
- Best regards, 
             ~ Alex

**Comment by axtloss** [comment]
first off:
> This guy commented bad about me, and the comment was left as it is, but when someone defended me, you minimized the comment. This shows what a stupid behavior you have.

I did hide that comment, it says the same thing as the other comment, not sure why it isn't working for you

Second: <br> 
yes, I did say the n word, twice, I'm not proud of that and I should've never done it, you however, said it many times and don't seem to be ever stopping. <br>
also, having black & LGBT friends (tomboys aren't always LGBT by the way), doesn't change the fact that you made jokes about gay people. You included a transphobic slur in piri in the pearos release **after** we discussed it. It doesn't make it better that you thought it's "just a meme". You still shared a transphobic meme in dms and laughed at it.
<br>
> if I put "straight" in my Instagram bio, I would be cancelled by the LGBT people, but if people from LGBT put the rainbow flags in bio, they are somehow "superior"

Yeah fun fact: Straight people don't get discriminated, LGBT people do, it's the same thing as saying "all lives matter".

> Every logo belongs to it's author, and what you did was: get the logo from me, get neofetch from GitHub.
according to your logic forks don't belong to the person that forked the project, so your piri must be mine, right?

I took your logo with **your permission** and modified the neofetch (it's called forking btw), so I **do** own that modified neofetch because it's a fork of witch I have taken the logo with your permission.
<br>
 > Why did you assume my gender? (that question is with humouristic purposes, I go by hell/boy. Jk again, I go as he/him. For real

 So you say you aren't transphobic, yet you make a transphobic joke(ignoring the fact that jokes don't belong in a serious discussion)? <br>
 > This was an example of convos you had with me. Did you tell me that you are trans? no

 BREAKING NEWS: PEOPLE DON'T IMMEDIATELY KNOW WHAT GENDER/SEXUALITY THEY ARE
 > I would not talk about the nword, "amy"

 WOOOOOOOOOOOW epic transphobia :+1: thanks for not accepting my new name. But noooo, you're totally not transphobic, sorry for assuming that.
 <br>
> More things you did: care too much about my financial situation. Why? Is your financial situation?

I never said anything about your financial situation, I just showed that you don't seem to be able pay your host, I picked up what I heard from others. <br>
> It is EXTREMLY stupid making a gh repo saying not to use a linux distro

No it's not, if a distribution is really bad and people don't see that, it's totally worth it to make a repo showing people why not to use it. <br>
> Finally, keep your GenZ shit away from me, kid

what does genz have to do with that? You're genz too. <br>
> You DO NOT deserve being a distro developer if you say things like "don t use this distro", "that distro is bad", "ew <redacted> distro". Extremly bad and this shows what kind of person you are.

I can't have an opinion? That's interesting.

> You still have my best regads, since I cannot put my brain agains you, being 4 years < me

It's funny how you act like you're smarter than me, yet I'm the one who helped you with quite some stuff in pearos that you wouldn't have figured out, not to mention licensing stuff that you still don't seem to understand.
---------------------
I've completely had it with you, I'm closing the issue now since the one thing I learned while working with you is that arguing with you can be like talking to a wall. You're a transphobe, you make transphobic jokes in a text saying you're not transphobic. You make claims that are objectively wrong.
Best regards,
amy (that's my name, deal with it).

**Comment by alxb421** [comment]
> I did hide that comment, it says the same thing as the other comment, not sure why it isn't working for you

It is working now

> yes, I did say the n word, twice, I'm not proud of that and I should've never done it, you however, said it many times and don't seem to be ever stopping.

So we both said the nword, as a joke. Did I stopeed? yes

> I took your logo with your permission and modified the neofetch (it's called forking btw), so I do own that modified neofetch because it's a fork of witch I have taken the logo with your permission.

And I removed your name cause people thought that you AND me made pearOS

> also, having black & LGBT friends (tomboys aren't always LGBT by the way), doesn't change the fact that you made jokes about gay

I m not proud of the jokes that I made/ laughed at, BUT as long as YOU accepted to share the same sense of humour, you should not complain. We **agreed** that those are only JOKES between us, with **no actual harm intensions**

> (...) if a distribution is really bad and people don't see that (...)

If a distribution is **really** bad, people would SEE that. I don t quite find what s the correlation between my jokes and the actual distro.

> BREAKING NEWS: PEOPLE DON'T IMMEDIATELY KNOW WHAT GENDER/SEXUALITY THEY ARE

alright, but in THAT time you agreed with me on those jokes

>WOOOOOOOOOOOW epic transphobia 👍 thanks for not accepting my new name. But noooo, you're totally not transphobic, sorry for assuming that.

Put the "amy" name in quotes because I usually go with legal name (which is ali). I do accept your name tho, as you can see, I used it many times in my (comment???)

> what does genz have to do with that? You're genz too.

I don t behave like a Gen Z

> just showed that you don't seem to be able pay your host

ok and...? What if I am not able to pay? Would you give me money?

> I can't have an opinion? That's interesting.

Ofcourse you CAN have opinions, but as long as you are making drama bout someone's distro, this shows how "small" you are

> It's funny how you act like you're smarter than me

Never said that


> some stuff in pearos that you wouldn't have figured out

I said Thank You a lot of times


> not to mention licensing stuff that you still don't seem to understand

In my files( downloaded files) it mentions the GPL v3 license. You changed licenses AFTER I forked it


> You're a transphobe

I am not, I support all people in exact same way


> that's my name, deal with it

Of course I accept your name. I used it a lot of times, even in convos.

**Comment by axtloss** [comment]
> Of course I accept your name. I used it a lot of times, even in convos

When? The first time I used my name with you was in an email to which you never responded to. 

> Put the "amy" name in quotes because I usually go with legal name (which is ali).

https://health.clevelandclinic.org/deadnaming/ please, for the love of god, educate yourself on this. You do not know how much I hate hearing that name. It's never ok to call a trans person who have chosen a new name with their deadname.

**Comment by alxb421** [comment]
![image](https://user-images.githubusercontent.com/72302254/153059958-704ef619-b435-496c-b91f-0c1439d690df.png)

Found this on the internet. I just wanted to ask bout it

**Comment by alxb421** [comment]
> an email to which you never responded to

I don t check the domain emails often.

> educate yourself on this.

Alright so in court, I should say amy, if it s the case of a lawsuit?
How do they identify you? 
In social maner, I accept your name, I don t give a fuck what is it, I just call you by that
But in a legal manel, I cannot call you by the new name. If you get a speed ticket... What name you give to that authority?

As I said, I have no harm intended. Are real life examples - please, leave out of the internet. Internet is a wild place, amy

**Comment by MikuOcta** [comment]
Hi!!!! Here for the tea, don't mind me UwU

**Comment by jaasio** [comment]
https://user-images.githubusercontent.com/54361578/153062784-cbcb32fe-e160-4951-92a0-301b957b25d3.MOV
🥺🥺🥺

**Comment by MikuOcta** [comment]
This conversation is leading literally nowhere. The fact that you are arguing even if the events were in the past shows what a childish nature you have. Plus, we went on different paths, Alex. You with your projects, us with ours. Plus, even if Amy was a gender before and now she is another is not a reason for you to complain about her name and "legal sex". It depends on what are inside, not outside, and even if you are +4 years than Amy, you still have a mind of a 2 year old, not understanding simple concepts. Sorry Alex, but everytime we cross paths, you just show what kind of a person you are.

**Comment by alxb421** [comment]
> This conversation is leading literally nowhere. The fact that you are arguing even if the events were in the past shows what a childish nature you have. Plus, we went on different paths, Alex. You with your projects, us with ours. And not only that, even if Amy was a gender before and now she is another is not a reason for you to complain about her name and "legal sex". It depends on what are inside, not outside, and even if you are +4 years than Amy, you still have a mind of a 2 year old, not understanding simple concepts. Sorry Alex, but everytime we cross paths, you just show what kind of a toxic person you are.

Yes, Miku. I didn't complain bout her and her gender. I complained 'bout her behaviour switch.

"Every time we cross paths", well Amy made the mistke of making a GitHub repo reopening things from the past, I am just defending muself.

"You with your projects, us with ours" I totally agreed, that is why I support CrystalUX but as I see, Amy does not support pear and she wants to put me down.

"the events were in the past shows what a childish nature you have": did I made this gh repo? Did I opened things from the past? I am 100% sure that I have the right to defend myself.

"what kind of a toxic person you are." As I said lots of times, I am DEFENDING myself. Amy attacked me. I wouldn't say anything 'bout this subject, but the fact that Amy decided to reopen it, shows her mental age.

You told me long ago that you are bisexual. Did I care? Did I told you anything? Was I against you and your sexuality? Nope. I was still helping you, answering your questions, supporting you. That (imo) clearly shows that I do not have anything against someone's sexuality. 

Stop defending Amy, because she was definitely not silent and the fact that she made a gh repo mentioning things from the past is pure childish behaviour.

Please, show me when did I opened things from the past instead of defending. You won't be able to because Amy started this bullshit against me

We (me and Amy) both did wrong things, and I decided to support her project and forgetting ehat happened in pearOS era. What is wrong bout this, Miku? As I talked to you on discord, you seem pretty open-minded and you understand a lot more things than Amy. Now, I would want to think about that in a impartially way. What we both did, who started this shit, and finally, come up with your final thoughts.

Cannot accuse someone for a thing that I did (i.e. amy accused me for saying nword while she did that too).

Again, the way we used the n word was to joke, and fun-talk without offending anyone(it was a private convo, right? Who would be offended if they cannot see the **private** conversation. *unless someone make it public without my legal agreement, which can lead to a lawsuit* quote(translated from Ro -> En):
```
Art. 71 - The right to privacy
(1) Everyone has the right to respect for his or her privacy.

(3) It is also forbidden to use, in any way, the correspondence, manuscripts or other personal documents, as well as the information from the private life of a person, without his consent or without respecting the limits provided in art. 75.

Art. 72 - The right to dignity
(1) Everyone has the right to respect for his dignity.
(2) It is forbidden any harm brought to the honor and reputation of a person, without his consent or without respecting the limits provided in art. 75.

Art. 74 - Violations of privacy
Subject to the application of the provisions of art. 75, may be considered as infringements of privacy:


i) the dissemination or use of correspondence, manuscripts or other personal documents, including data on the domicile, residence, and telephone numbers of a person or members of his family, without the consent of the person to whom they belong or who, as the case may be, has the right to dispose of them.
```

Just to know 🤷🏿‍♂️
~ Alex
